.github/workflows/verify-platform.yml

```diff
@@ -61,7 +62,8 @@ jobs:
        JAVA_HOME: ${{ env.JAVA_HOME_17_X64 }}
       working-directory: 'platform'
       run: >- 
-        mvn -U -V -e -B -ntp
+        mvn -U -V -e -B -ntp -X
```

Comment:
Will this be reverted or is this by intention? The verification here should really not fail for any baseline comparison as we never change any code.

Reply:
Same as above.